comparison.tex

```diff
@@ -0,0 +1,36 @@
+\section{Comparison}
+
+We present a few alternatives for performing proof of burn, all of which are captured by our burn protocol definition.
+
+\noindent
+\textbf{OP\_RETURN.}
+Bitcoin natively employs a mechanism for burning money using a specialized
+opcode called OP\_RETURN~\cite{bartoletti2017analysis}. Unfortunately,
+creating an OP\_RETURN proof-of-burn transaction is complicated and not
+user-friendly. However, it does benefit the Bitcoin network by allowing the UTXO
+to be pruned, at the cost of not being uncensorable.
```

Comment:
Mention lack of flexibility here: It requires scripting capabilities, which is not something every cryptocurrency employs.

Reply:
Lack of flexibility is mentioned in the comparison part before the table for both OP_RETURN and P2SH OP_RETURN.